csm_web/frontend/src/components/Policies.tsx

```diff
@@ -177,22 +177,22 @@ export default class Policies extends React.Component {
               {" "}
               <b> {faq.body.p7_b} </b>{" "}
             </p>
-            <p>
+            <span>
               {faq.body.p8}
               <ul>
                 <li> {faq.body.li1} </li>
                 <li> {faq.body.li2} </li>
               </ul>
-            </p>
-            <p>
+            </span>
```

Comment:
Is there any particular reason why the body text needs to be grouped with the list text like this? I feel an easier solution is to just move the closing tag before the `<ul>` tags.

Reply:
oo u are very right! updated